docs/specs/xref.yml

```diff
@@ -2162,3 +2162,69 @@ outputs:
   .errors.log: |
     {"message_severity":"warning","code":"missing-attribute","message":"Missing required attribute: 'name'.","file":"docs/TOC.md"}
     {"message_severity":"warning","code":"xref-not-found","message":"Cross reference not found: 'b'.","file":"docs/TOC.md"}
+---
+# files to be redirected must be excluded from xref map
+noSingleFile: true
+repos:
```

Comment:
This can be simplified to `inputs:`, since we do not have multiple commits or `.openpublishing.publish.config.json`